flowz/channels/tools.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+import sys
```

Comment:
Would you mind adding a docstring at the top of the module to indicate that this is for helpers rather than for expanding the core functionality of the channel classes?